pkg/deploy/registry/rest.go

```diff
@@ -17,6 +17,41 @@ var (
 	ErrUnknownDeploymentPhase = errors.New("unknown deployment phase")
 )
 
+// WaitForRunningDeployer waits until the specified deployer pod name is
+// running.
+func WaitForRunningDeployer(pn kclient.PodsNamespacer, namespace, name string, timeout time.Duration) (bool, error) {
```

Comment:
@kargakis I think this should be some generic helper (if it is not already), something like "WaitForPodRunning()" or "WaitForPodCondition()"... any idea if something like that exists?